src/arkit.cs

```diff
@@ -298,6 +356,38 @@ interface ARPointCloud : NSSecureCoding {
 		IntPtr GetRawIdentifiers ();
 	}
 
+	[NoWatch, NoTV, NoMac, iOS (11,3)]
+	[BaseType (typeof(NSObject))]
+	interface ARReferenceImage : NSCopying, NSSecureCoding {
```

Comment:
this is also not clear it can be created with `init`, the 2 .ctors don't accept `nil` images
easier to add it later than removing it (breaking changes)